### DISCUSSION TITLE
disabling the RefreshContainer refresh function.

### DISCUSSION THREAD

**Discussion by Marc-Antoine-Soucy** [unoplatform/uno#13628] [Q&A]
In an app, I have a RefreshContainer which is over a listview, like this:
```
<RefreshContainer>
  <ListView ... 
                  .../>
</RefreshContainer>
```
The problem I have is that sometimes, I want to enable drag and drop in the ListView, and the Refresh container sometimes makes it harder to pick an element in the list and drag it. So, I would like to make the RefreshContainer stop trying to refresh my list, when drag and drop is enabled. 

The only simple way I found to do this was by Collapsing the refreshcontainer when drag and drop is enabled and making another list with the drag and drop visible. Like this:
```
<RefreshContainer Visibility={Binding CanDrag, Converter={StaticResource TrueToCollapse}>
  <ListView ... 
                  .../>
</RefreshContainer>

<ListVIew Visibility={binding CanDrag, Converter={StaticResource TrueToVisible}
  ..
  ../>
```
I tried separating the RefreshContainer from the listview, but either the Refreshcontainer will block the listview's button from being clicked, or the listview will block the refresh container. Is there a better way to do this?

**Comment by jeromelaban** [comment]
@MartinZikmund may know
